aptos-move/framework/aptos-framework/sources/configs/staking_config.move

```diff
@@ -35,24 +46,36 @@ module aptos_framework::staking_config {
         allow_validator_set_change: bool,
         // The maximum rewards given out every epoch. This will be divided by the rewards rate denominator.
         // For example, 0.001% (0.00001) can be represented as 10 / 1000000.
-        rewards_rate: u64,
-        rewards_rate_denominator: u64,
+        _rewards_rate: u64,
```

Comment:
You can't change the struct names unfortunately as that would fail the compatibility test (upgrade would fail). You can leave comments to document that these should be deprecated.